moveos/moveos-client/src/client.rs

```diff
@@ -0,0 +1,194 @@
+struct SendTxnRequest {
+    sender_address: String,
+    public_key: Vec<u8>,
+    program: Vec<u8>,
+    sequencer_number: u64,
+    expiration_time: u64,
+    signature: Vec<u8>,
+}
+
+struct SendTxnResponse {
```

Comment:
must add order_hash: Vec<u8> as return value. It's element in order accumulator, and sequencer must give it.

It's sha256<txn_hash, order>